packages/syft/src/syft/core/tensor/autodp/row_entity_phi.py

```diff
@@ -187,7 +189,7 @@ def __mul__(  # type: ignore
                     )
                 else:
                     raise Exception(
-                        f"Tensor dims do not match for __sub__: {self.child.shape} != {other.shape}"  # type: ignore
+                        f"Tensor dims do not match for __sub__: {getattr(self.child, 'shape', None)} != {other.shape}"
```

Comment:
Child probably shouldn't have a shape property but at least this can safely try to get it.